aws/aws.go

```diff
@@ -511,6 +527,11 @@ func NukeAllResources(account *AwsAccountResources, regions []string) error {
 						continue
 					}
 
+					if options.IgnoreErrors {
+						logging.Logger.Warnf("Encountered error nuking resource type %s in region %s, but --ignore-errors flag is passed in so continuing.", resources.ResourceName(), region)
+						allErrors.Add(RegionResourceInfo{region: region, resource: resources.ResourceName()}, err)
```

Comment:
Handling the error at this level means that this particular batch will only be partially processed, right? E.g., Perhaps it successfully deletes ELBs, but hits an error half way through deleting ASGs, so for this batch, it does not finish the other half of the ASGs, or any other resources after the ASG, such as EC2s, S3 buckets, etc?